lua/pl/lapp.lua

```diff
@@ -198,6 +210,13 @@ function lapp.process_options_string(str,args)
 
     usage = str
 
+    for _,a in ipairs(args) do
```

Comment:
This should be `ipairs(arg)`, not `ipairs(args)`, otherwise parsing default command-line arguments is broken.